src/Components/Components/src/Microsoft.AspNetCore.Components.csproj

```diff
@@ -10,6 +10,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <Reference Include="Microsoft.AspNetCore.Authorization" />
```

Comment:
I don't have a better idea right now, but are we really OK with baking this in at the lowest level. We're saying **all** components present and future have access to the ASP.NET Authorization library

Reply:
Yep, I agree with both the uneasiness and the conclusion that this is the best outcome currently. 

Considering that all the Components/Blazor packages have `AspNetCore` in their name, it doesn't really change anything to reference `Microsoft.AspNetCore.Authorization` as compared with if we had `Microsoft.AspNetCore.Components.Authorization`.

Reply:
Yeah, this makes me a bit sad on the inside. I'm wondering if we could use some of the infrastructure for single file publish coming in 3.0 to separate the core of the platform from the baked-in components that we give you into two assemblies and simply ILMerge them at publish time.